puan/logic/plog/__init__.py

```diff
@@ -1582,14 +1582,55 @@ class Equal(All):
         Examples
         --------
         Meaning exactly two of x, y and z.
+            >>> Equal(2, list("xyz"), variable='A')
+            A: +(VAR658adc74c6913fb83b42c3968866f4bdb967fcad34692fc71d3de5f9a94b6970,VARe4568f4a0e4e55b7e3afa57b3527153272b53fde10f6e292b510a5c3bf797d3d)>=2
             >>> Equal(2, list("xyz"), variable='A').propositions
-            [A: +(x,y,z)>=2, A: -(x,y,z)>=-2]
+            [VAR658adc74c6913fb83b42c3968866f4bdb967fcad34692fc71d3de5f9a94b6970: -(x,y,z)>=-2, VARe4568f4a0e4e55b7e3afa57b3527153272b53fde10f6e292b510a5c3bf797d3d: +(x,y,z)>=2]
     """
     
```

Comment:
Do you need to add here that you added the methods to/from json?

        Methods
        -------
        from_json
        to_json
        
        And: Do you need the method from_list?